serverless/architecture/serverless-serving-architecture.adoc

```diff
@@ -0,0 +1,27 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-serving-architecture"]
+= Knative Serving architecture
+:context: serverless-serving-architecture
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+Knative Serving supports deploying and managing serverless applications, by providing a set of objects as Kubernetes custom resources (CRs) that define and control the behavior of serverless workloads on an {product-title} cluster.
+
+These CRs are building blocks to address complex use cases, for example:
+
+* Rapidly deploying serverless containers.
+* Automatically scaling pods.
+// TODO: Add other use cases, more advanced tutorials?
+
+[id="serverless-serving-architecture-custom-resources"]
+== Knative Serving custom resources
+// TODO: Request a good Serving arch diagram
+
+Configuration:: The `configuration.serving.knative.dev` resource maintains the desired state for your deployment. It provides a clean separation between code and configuration. Modifying a configuration creates a new Revision.
+
+Revision:: The `revision.serving.knative.dev` resource is a point-in-time snapshot of the code and configuration for each modification made to the workload. Revisions are immutable objects and can be retained for as long as necessary.
+
+Service:: The `service.serving.knative.dev` resource automatically manages the life cycle of your workload. It controls the creation of other objects to ensure that your application has a Route, a Configuration, and a new Revision for each change to the Service. Services enable traffic routing to your chosen Revision.
```

Comment:
In case you move this to the top:

```suggestion
Service:: The `service.serving.knative.dev` resource automatically manages the life cycle of your workload. It creates a Configuration and a Route as explained below to make sure the application is deployed and reachable via the network. Most interactions should be done through Services.
```